packages/yew/src/html/component/scope.rs

```diff
@@ -512,17 +512,19 @@ mod feat_csr {
             root: BSubtree,
             parent: Element,
             next_sibling: NodeRef,
-            node_ref: NodeRef,
+            internal_ref: NodeRef,
             props: Rc<COMP::Properties>,
         ) {
             let bundle = Bundle::new();
-            node_ref.link(next_sibling.clone());
+            internal_ref.link(next_sibling.clone());
+            let stable_next_sibling = NodeRef::default();
```

Comment:
Do we need to create a linked `NodeRef` only for it to be linked again?

Reply:
`internal_ref` is linked to `next_sibling` to track the position while the component doesn't yet exist.

`stable_next_sibling` is *also* linked to `next_sibling`, and a clone of `stable_next_sibling` is passed down the tree and thus "automatically" updated when `stable_next_sibling` is linked to an updated `next_sibling` later. The alternative would be to traverse the tree all the way down when updating `next_sibling`, which is more expensive - not even considering additional code size.

Overall, `next_sibling` is linked from two independent other refs, but there's no chain, if that's what you mean.